modules/@ergonode/attributes/src/components/Lists/AttributesListElement.vue

```diff
@@ -83,15 +86,20 @@ export default {
             return this.item.label || `#${this.item.code}`;
         },
         typeIconComponent() {
-            const extendedIcon = this.$getExtendedComponents('@Attributes/components/Lists/AttributeListElement/Icon');
+            const icon = this.$getExtendedFormByType({
+                key: '@Attributes/components/Lists/AttributeListElement/Icon',
+                type: this.item.type,
+            });
 
-            if (extendedIcon && extendedIcon[this.item.type]) {
-                return extendedIcon[this.item.type];
+            if (!icon.length) {
+                return [
+                    {
+                        component: () => import('@Core/components/Icons/Menu/IconAttributes'),
+                    },
+                ];
             }
 
-            return {
-                component: () => import('@Core/components/Icons/Menu/IconAttributes'),
-            };
+            return icon;
```

Comment:
Why is it checking icon.length? Is It array? String? If it will be undefined, numeric it will crash app.

Reply:
this is always array type.